bench/utils.py

```diff
@@ -1171,6 +1171,15 @@ def migrate_env(python, backup=False):
 		raise
 
 
+def is_dist_editable(dist):
+	"""Is distribution an editable install?"""
+	for path_item in sys.path:
+		egg_link = os.path.join(path_item, dist + '.egg-link')
```

Comment:
.egg-link file exists in site-packages if installed in editable mode. eg: `/Library/Python/2.7/site-packages/bench.egg-link`